packages/catalog-process/src/services/catalogService.ts

```diff
@@ -515,7 +514,7 @@ export function catalogServiceBuilder(
         eserviceId,
         descriptorId,
         authData,
-        copyFile: fileManager.copy,
+        copyFile: () => Promise.resolve(""), // TODO unmock and use FileManager
```

Comment:
Leaving it unmocked breaks the tests now that the file manager is implemented, since it tries to copy a file that does not exist - since this PR is already pretty big, I decided to mock the file manager usages except for `deleteDocument` (to have at least one full stack method tested in this PR). 

I will unmock & test the other file manager calls in further PRs (tracked here: https://pagopa.atlassian.net/browse/IMN-262)

(see also PR description)